vpplink/vxlan.go

```diff
@@ -23,6 +23,34 @@ import (
 	"github.com/projectcalico/vpp-dataplane/vpplink/types"
 )
 
+func (v *VppLink) ListVXLanTunnels() ([]*types.VXLanTunnel, error) {
```

Comment:
Might be better to return `[]types.VXLanTunnel` instead of a list of pointers. 
Although I'm guilty of making it wrong in ListIPIP & cie